snap/local/hooks/Makefile

```diff
@@ -0,0 +1,24 @@
+GO=go
+HOOKS=cmd/configure/configure cmd/install/install
+
+tidy:
+	go mod tidy
+
+build: tidy $(HOOKS)
+
+cmd/configure/configure:
+	$(GO) build -o $@ ./cmd/configure
+
+cmd/install/install:
+	$(GO) build -o $@ ./cmd/install
+
+# TODO: fixup unit testing
+test:
+	$(GO) test -coverprofile=coverage.out ./...
+	$(GO) vet ./...
+	gofmt -l .
+	[ "`gofmt -l .`" = "" ]
+
+clean:
+	rm -f $(HOOKS)
+
```

Comment:
Instead of separate comments and you having to piece things together, I felt it might be easier to just show the final result:


Changes:
1. Move `build` above `tidy` as `build` should be the default target when just running `make` with no target specified.
2. Building of `$(HOOKS)` can be simplified to reduce redundancy
3. Specify [`.PHONY` targets](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html)

```make
GO=go
HOOKS=cmd/configure/configure cmd/install/install

.PHONY: tidy build test clean

build: tidy $(HOOKS)

tidy:
	go mod tidy

$(HOOKS):
	$(GO) build -o $@ ./$(dir $@)

# TODO: fixup unit testing
test:
	$(GO) test -coverprofile=coverage.out ./...
	$(GO) vet ./...
	gofmt -l .
	[ "`gofmt -l .`" = "" ]

clean:
	rm -f $(HOOKS)
```

Reply:
made change